Ghidra/Features/Base/src/main/java/ghidra/app/script/GhidraScriptProvider.java

```diff
@@ -29,6 +30,9 @@
 public abstract class GhidraScriptProvider
 		implements ExtensionPoint, Comparable<GhidraScriptProvider> {
 
+	private static final Pattern BLOCK_COMMENT_START = Pattern.compile("/\\*");
+	private static final Pattern BLOCK_COMMENT_END = Pattern.compile("\\*/");
```

Comment:
Shouldn't these be moved to `JavaScriptProvider` instead of the base class?  The base class behavior should be _no_ block comment checking rather than the fallback to Java.